messaging/src/main/java/org/axonframework/eventhandling/deadletter/DeadLetteringEventHandlerInvoker.java

```diff
@@ -196,6 +260,9 @@ public static class Builder extends SimpleEventHandlerInvoker.Builder<Builder> {
         private EnqueuePolicy<EventMessage<?>> enqueuePolicy = (letter, cause) -> Decisions.enqueue(truncated(cause));
         private TransactionManager transactionManager;
         private boolean allowReset = false;
+        private boolean cacheEnabled = false;
+        private Cache cache = NoCache.INSTANCE;
```

Comment:
As discussed offline, let's replace the `Cache` for a simple `ConcurrentHashMap`.

Reply:
Good suggestion, changed.